net-p2p/litecoind/litecoind-0.18.1.ebuild

```
@@ -0,0 +1,90 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DB_VER="4.8"
+
+inherit autotools db-use flag-o-matic systemd
+
+MY_PV="${PV/_/-}"
+MY_PN="litecoin"
+MY_P="${MY_PN}-${MY_PV}"
+
+DESCRIPTION="The offical daemon to run your own (full) Litecoin node"
+HOMEPAGE="https://litecoin.org/"
+SRC_URI="https://github.com/${MY_PN}-project/${MY_PN}/archive/v${MY_PV}.tar.gz -> ${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="MIT ISC GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="logrotate upnp +wallet"
```

```suggestion
IUSE="upnp +wallet"
```
and remove src_install usage